versioned_docs/version-1.20.6/datastorage/saveddata.md

```diff
@@ -22,19 +22,36 @@ Any `SavedData` is loaded and/or attached to a level dynamically. As such, if on
 For example, if a SD was named "example" within the Nether, then a file would be created at `./<level_folder>/DIM-1/data/example.dat` and would be implemented like so:
 
 ```java
-// In some class
-public ExampleSavedData create() {
-  return new ExampleSavedData();
-}
-
-public ExampleSavedData load(CompoundTag tag, HolderLookup.Provider lookupProvider) {
-  ExampleSavedData data = this.create();
-  // Load saved data
-  return data;
+// In some saved data implementation
+public class ExampleSavedData extends SavedData {
+
+    // Create new instance of saved data
+    public static ExampleSavedData create() {
+        return new ExampleSavedData();
+    }
+
+    // Load existing instance of saved data
+    public static ExampleSavedData load(CompoundTag tag) {
+        ExampleSavedData data = ExampleSavedData.create();
+        // Load saved data
+        return data;
+    }
+
+    @Override
+    public CompoundTag save(CompoundTag tag) {
```

Comment:
`HolderLookup.Provider` was added to these methods in 1.20.5, so this should still have parameter.

Reply:
Done